src/children.js

```diff
@@ -34,6 +36,28 @@ export function paths (node) {
 	return children;
 }
 
+/**
+ * Replaces a child node with a new node, and updates the parent node and parent pointers
+ * @param {object} child 
+ * @param {object} newChild 
+ */
+export function replace (child, newChild) {
+	const parentPath = parents.path(child);
+	if (parentPath) {
+		const {property, index, node: parent} = parentPath;
```

Comment:
This will error if `parentPath` is `null` or `undefined`

Reply:
How will it error? If it is null or undefined then the node is a root or doesn't have parents set, meaning replacement is not possible, what were you thinking should happen in this case?